specification/alertsmanagement/resource-manager/readme.md

```diff
@@ -41,10 +41,22 @@ directive:
 title: AlertsManagementClient
 description: AlertsManagement Client
 openapi-type: arm
-tag: package-preview-2021-08
+tag: package-2021-08
 ```
 
 
+
+### Tag: package-2021-08
+
+These settings apply only when `--tag=package-2021-08` is specified on the command line.
+
+```yaml $(tag) == 'package-2021-08'
+input-file:
+  - Microsoft.AlertsManagement/stable/2021-08-08/AlertProcessingRules.json
+  - Microsoft.AlertsManagement/preview/2019-05-05-preview/AlertsManagement.json
+  - Microsoft.AlertsManagement/preview/2019-05-05-preview/SmartGroups.json
```

Comment:
```suggestion
  - Microsoft.AlertsManagement/preview/2019-05-05-preview/SmartGroups.json
  - Microsoft.AlertsManagement/stable/2019-06-01/SmartDetectorAlertRulesApi.json
```

Reply:
@msyyc according to the smart detector team, that api was never published in any sdk.
is there a way to validate this? Does it matter, or is it better to have support in the sdk even if its not tested or used?